lib/auth/sso_mfa.go

```diff
@@ -154,6 +154,9 @@ func (a *Server) upsertSSOMFASession(ctx context.Context, user string, sessionID
 			Scope:      ext.Scope,
 			AllowReuse: ext.AllowReuse,
 		},
+		Payload: &mfatypes.SessionIdentifyingPayload{
+			SSHSessionID: sip.GetSshSessionId(),
+		},
```

Comment:
Suggestion: only set this if `sip != nil`, so the Payload itself is nil if not needed?